tools/link.py

```diff
@@ -1955,9 +1958,25 @@ def phase_embind_emit_tsd(options, wasm_target, js_syms):
     node_args += shared.node_memory64_flags()
   # Run the generated JS file with the proper flags to generate the TypeScript bindings.
   out = shared.run_js_tool(outfile_js, [], node_args, stdout=PIPE)
+  settings.restore(original_settings)
+  return out
+
+
+@ToolchainProfiler.profile_block('embind emit tsd')
+def phase_embind_emit_tsd(options, wasm_target, js_syms):
+  logger.debug('emit tsd')
+  out = run_embind_gen(wasm_target, js_syms, {'EMBIND_AOT_JS': False})
   write_file(
     os.path.join(os.path.dirname(wasm_target), options.embind_emit_tsd), out)
```

Comment:
Maybe put the filename in its own variable to avoid splitting this line like this?